server/build/Dockerfile.fips

```diff
@@ -0,0 +1,83 @@
+# First stage - FIPS dev image with dependencies for building
```

Comment:
@esarafianou, updated this Dockerfile to use multiple stages so we don't ship with the `-dev` version of the Chainguard image.